cloud-governance-mcp/app.py

```
@@ -0,0 +1,770 @@
+                result = loop.run_until_complete(fetch_mcp_tools_async())
+            finally:
+                loop.close()
+        except Exception as e:
+            error = e
+
+    thread = threading.Thread(target=run_in_thread)
+    thread.start()
+    thread.join()
+
+    if error:
+        st.error(f"❌ Failed to connect to MCP server")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)